test/library/packages/cuBLAS/test_cublas1.compopts

```diff
@@ -0,0 +1 @@
+-L/usr/local/cuda-10.2/lib64 -lcudart -lcublas
```

Comment:
I wonder if we could infer these flags from a `pkg-config` call instead to avoid hard-coding paths and versions.